### ISSUE TITLE
Consider use @Optional annotation to replace @Required

### ISSUE THREAD

**Issue by Muyangmin** [workarounds/bundler#15]
At least in current version,  if `@Required` annotation is omitted, this field itself is required in fact. So we use `@Required` annotation **only when it is NOT required**, so why not change it to `@Optional` annotation? I think it's more simple and obvious.

``` Java
@Arg
@Required(false)
@Optional
String bestWishes;
```

**Comment by madki** [comment]
@Muyangmin Yup, that sounds good. Been meaning to do that since a while, but I have no time on my hands currently. A pull request would be greatly appreciated if you're up for it :)
